test/KregielniaTests.cpp

```diff
@@ -1,11 +1,46 @@
 #include <gtest/gtest.h>
 #include "Kregielnia.hpp"
+#include "StringToVectorConverter.hpp"
+#include <string>
 
 struct KregielniaTests : public ::testing::Test
 {
 };
 
+int sumVector(std::vector<int> vec){
+    int sum = 0;
+    for(auto i : vec)
+        sum += i;
+    return sum;
+}
+
 TEST_F(KregielniaTests, assertThatTravisCiBuildStatusIsPass)
 {
     ASSERT_TRUE(true);
 }
+
+TEST_F(KregielniaTests, pojedynczaCyfra)
```

Comment:
Test names should be in English and should be more meaningful

Reply:
Done